buzsaki_lab_to_nwb/huszar_hippocampus_dynamics/questions.md

```diff
@@ -0,0 +1,36 @@
+
+## To ask the authors
+
+## Todo
+* In the paper the birthdates are marked as: E13.5, E14.5, E15.5, and E16.5
+This means embrionic age (E) apparently. How does it track with the recent changes on age representation on nwb
```

Comment:
This is days of conception, my term birthdates there is wrong. The relevant section on the paper:

```
Early morning of the
next day was considered embryonic (E) age 0.5. Time pregnant mice and their offspring were
kept on a regular light-dark cycle. Electrode-implanted adult mice (3-6 months) were housed
individually on a reverse light-dark cycle.
```

Reply:
Ahh.. hmm.

Soo, there's a constant time offset to get it back into age relative to birth?

Otherwise, I suppose the age reference for this could use the new value 'gestational'